observability/cluster_observability_operator/cluster-observability-operator-release-notes.adoc

```diff
@@ -43,6 +43,17 @@ The following table provides information about which features are available depe
 | 1.1           | 4.18+              | ✔                     | ✔         | ✔                     | ✔          | ✔
 |===
 
+[id="cluster-observability-operator-1-1-known-issues_{context}"]
+== Known issues
+
+These are the known issues in {coo-full} 1.1.0:
+
+* `observability-operator` and `perses-operator` pods in `CrashLoopBackOff` due to `OutOfMemory` error, after upgrading from {coo-full} 1.0.
```

Comment:
```suggestion
* `observability-operator` and `perses-operator` pods get stuck in a  `CrashLoopBackOff` state due to `OutOfMemory` errors, after upgrading from {coo-full} 1.0.
```

Reply:
not sure about the "get stuck" bit, so will leave that out, but will add other bits